assets/css/style.css

```diff
@@ -171,60 +172,6 @@ footer {
 
   color: #cacdd2;
 }
-/* Newsletter Section Styles */
```

Comment:
why u have deleted the newsletter commits ???
For this changes the newsletter can't be seen in the website home page ...please try to fix this ASAP , otherwise I can't merge your PR .....all the other changes are good